templates/.prettierignore

```diff
@@ -0,0 +1,10 @@
+*-lock.*
+*.lock
+*.log
+
+/tsconfig.json
+
+build/
+dist/
+
+/worker-rust/target/
```

Comment:
Can we move these to the top-level .prettierignore?

Reply:
I wasn't sure if they should be scoped or top-level I definitely not against moving them up.